aws/inputseeders/aws/masterdata.json

```diff
@@ -946,6 +956,127 @@
                     }
                 }
             }
+        },
+        "_awswin" : {
```

Comment:
I think this is probably better as windows rather than awswin. awslinux is an actual distribution. Where the Windows install is really windows.